Sources/Testing/Support/Graph.swift

```diff
@@ -236,9 +285,10 @@ extension Graph {
   /// intermediate to it, the graph is not modified. To add a value when none
   /// previously exists, use ``insertValue(_:at:intermediateValue:)``.
   ///
-  /// - Complexity: O(*m* + *n*) where *n* is the number of elements in
-  ///   `keyPath` and *m* is the number of children at the penultimate node in
-  ///   `keyPath`.
+  /// - Complexity: To get a value, O(*n*) where *n* is the number of elements
+  ///   in `keyPath`. To set a value, O(*m* + *n*) where *n* is the number of
+  ///   elements in `keyPath` and *m* is the number of children at the
+  ///   penultimate node in `keyPath`.
```

Comment:
Why include the "To **get** a value..." sentence in this `updateValue` method, if it's focused on setting a value?

Reply:
This was a pasting error. :)